org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/discovery/ZigBeeDiscoveryService.java

```diff
@@ -227,4 +249,17 @@ public void run() {
 
         scheduler.schedule(pollingRunnable, 10, TimeUnit.MILLISECONDS);
     }
+
+    private void parseAndSetCreateResultsOnlyDuringActiveScansConfigProperty(Map<String, Object> properties) {
+        if (properties != null) {
+            Object property = properties.get(CONFIG_PROPERTY_CREATE_RESULTS_ONLY_DURING_ACTIVE_SCANS);
+            if (property != null) {
+                if (property instanceof Boolean) {
+                    createResultsOnlyDuringActiveScans = (Boolean) property;
+                } else if (property instanceof String) {
+                    createResultsOnlyDuringActiveScans = Boolean.parseBoolean((String) property);
+                }
+            }
```

Comment:
I think we need to handle `property == null`. The property could have been removed after is was true. In that case `createResultsOnlyDuringActiveScans` should be set to true.

Reply:
Right, it has to be set to `false` to restore the default behavior. I have amended my commit to reflect this.